migrations/Version202608051201083025_parccTei.php

```
@@ -0,0 +1,42 @@
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202608051201083025_parccTei extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Update graphNumberLineInteraction with minor fixes';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)